copy/copy.go

```diff
@@ -1060,6 +1060,14 @@ func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, to
 			logrus.Debugf("Skipping blob %s (already present):", srcInfo.Digest)
 			bar := ic.c.createProgressBar(pool, srcInfo, "blob", "skipped: already exists")
 			bar.SetTotal(0, true)
+
+			// Throw an event that the layer has been skipped
+			if ic.c.progress != nil {
+				ic.c.progress <- types.ProgressProperties{
+					Event:    types.ProgressEventSkipped,
+					Artifact: blobInfo,
```

Comment:
Elsewhere (both in the progress bar code above, and in the event channel used when compressing/decompressing during copy), we use the original `srcInfo` data/digests; so it seems reasonable to do the same here.

Reply:
Makes sense, changed it to use `srcInfo` :+1: